### ISSUE TITLE
Migration impossible since bots are bound to the user creating them and not the room

### ISSUE THREAD

**Issue by MTRNord** [vector-im/element-integration-manager#57]
**Describe the bug**
I have a bot in my room created on an old (now) non-existent homeserver. I got the migration message, however there is now no migration possible unless I do it manually since the bots are bound to users not rooms and the people with permission within.

This also is an issue with orgs where people might leave.

**To Reproduce**
1. Create a bot (It probably works with old and new ones)
2. Loose access to the account you created within
3. You are stuck. -> Manual intervention needed. Integration manager cannot be used for the bot anymore.

**Expected behavior**
Bots in Rooms should be bound to the admins or other PL (preferable configurable) people in the room and be sharing permission. Not to the individuals itself. Individuals are not a constant within a room. Only rooms itself are.

**Screenshots**
Not applicable

**Client (please complete the following information):**
 - Matrix Client and Version: Element-Web 
 - Matrix Client Version: dcc0cb74fe67-react-75f31defd635-js-5f3e115545e1
 - Browser: Firefox
 - Browser Version: 111.0 (64-bit)

**Comment by Half-Shot** [comment]
The new bots are bound to the room (and require a PL of 50 to modify), so this isn't a problem going forwards. For the existing bots, where exactly are you getting stuck. Is the migration not visible to admins when you open the extensions manager? Are you unable to kick the bot?

**Comment by Mikaela** [comment]
In develop.element.io

1. Click integrations
2. Select feeds
3. It asks me for URL and Label

Nowhere I am told what are the existing feeds that have been added to the room by another user and I would like to just mass import them without having to copy-paste.

**Comment by MTRNord** [comment]
@Mikaela already basically explains my issue. While I roughly might know what the other person has set up for example a git repo for commit notifications I don't know exactly. People simply may be hard to teach, impossible to reach because they left the company or simply because I set the bot up years ago and changed my homeserver twice in that time. So I simply can't look it up.

**Comment by Half-Shot** [comment]
Right....My question was "Can **admins** of the room not see Go-NEB feeds by other users". It sounds like the answer is No, which sounds like a Hookshot bug.
